swiftui-project/ProductDetailsView.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  HomePageView.swift
+//  swiftui-project
+//
+//  Created by user238749 on 5/7/23.
+//
+
+import SwiftUI
+
+struct ProductDetailsView: View {
+    
+    var body: some View {
+       
+            VStack(alignment: .leading) {
+                HStack {
+                    
+                    Spacer()
+                    Text("Product details")
+                        .font(.title3)
+                       
+                    Spacer()
+                    Button(action: {}, label: {
+                        Image(systemName: "cart.fill")
+                            .font(.title2)
+                            .foregroundColor(Color.black)
+                    })
+                }
+                .padding(.horizontal)
+                
+                ZStack(alignment: .topTrailing) {
+                    Image("ProductDetails")
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                        .cornerRadius(10)
+                        .padding(.horizontal, 20)
+                    
+                    Button(action: {}, label: {
+                        HStack(spacing: 4) {
+                            Image(systemName: "checkmark.circle.fill")
+                                .foregroundColor(.green)
+                            Text("In stock")
+                                .foregroundColor(.black)
+                            
+                        }
+                        .padding(.horizontal, 10)
+                        .padding(.vertical, 10)
+                        .background(Color.white)
+                        .cornerRadius(26)
+                        .padding(.vertical,15)
+                        .padding(.horizontal, 30)
+                    })
+                }
+                
+                
+                VStack(alignment: .leading) {
+                    HStack(spacing:1){
+                        Text("Stargazer's Tea Set")
+                            .font(.title3)
+                            .bold()
+                        Spacer()
+                        Text("5")
+                            .font(.headline)
+                            .bold()
+                        Image(systemName: "star.fill")
+                            .font(.subheadline)
```

Comment:
why do you use font for image? May be you need to resize it in that case .resizable and .frame is just one of the ways to do it

Reply:
Changed to
 .resizable()
.frame(width: 12,height: 12)